heron/config/src/yaml/conf/aurora/heron.aurora

```diff
@@ -20,7 +20,7 @@ fetch_user_package = Process(
   cmdline = 'curl %s -o %s && tar zxf %s' % (heron_topology_jar_uri, topology_package_file, topology_package_file)
 )
 
-command_to_start_executor = '{{EXECUTOR_BINARY}} {{mesos.instance}} {{TOPOLOGY_NAME}} {{TOPOLOGY_ID}} {{TOPOLOGY_DEFINITION_FILE}} {{STATEMGR_CONNECTION_STRING}} {{STATEMGR_ROOT_PATH}} {{TMASTER_BINARY}} {{STMGR_BINARY}} "{{METRICSMGR_CLASSPATH}}" {{INSTANCE_JVM_OPTS_IN_BASE64}} "{{TOPOLOGY_CLASSPATH}}" {{thermos.ports[port1]}} {{thermos.ports[port2]}} {{thermos.ports[port3]}} {{SYSTEM_YAML}} {{COMPONENT_RAMMAP}} {{COMPONENT_JVM_OPTS_IN_BASE64}} {{TOPOLOGY_PACKAGE_TYPE}} {{TOPOLOGY_BINARY_FILE}} {{JAVA_HOME}} {{thermos.ports[http]}} {{SHELL_BINARY}} {{thermos.ports[port4]}} {{CLUSTER}} {{ROLE}} {{ENVIRON}} "{{INSTANCE_CLASSPATH}}" {{METRICS_YAML}} "{{SCHEDULER_CLASSPATH}}" "{{thermos.ports[scheduler]}}" {{PYTHON_INSTANCE_BINARY}} {{METRICSCACHEMGR_CLASSPATH}} {{thermos.ports[metricscachemgr_masterport]}} {{thermos.ports[metricscachemgr_statsport]}} {{IS_STATEFUL_ENABLED}} "{{CKPTMGR_CLASSPATH}}" {{thermos.ports[ckptmgr_port]}} {{STATEFUL_CONFIG_YAML}}'
```

Comment:
Since this change changes the aurora conf file and also adds new yaml conf file, it will break the current topology deployment process for Twitter's environment. Please coordinate with @huijunw @objmagic or me to make sure that the necessary changes inside twitter are also made before merging it